swagger/en/schemas/participant.yml

```diff
@@ -140,11 +116,3 @@ properties:
             description: "This player's placement in the match"
             minimum: 1
             maximum: 130
-          winPoints:
-            type: integer
-            description: "N/A"
-            deprecated: true
-          winPointsDelta:
-            type: number
-            description: "N/A"
-            deprecated: true

```

Comment:
Do we need to add something to the changelog for these?

Reply:
It seemed like, historically, we've removed things from the docs when they were removed from the API.